cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -252,10 +252,12 @@ features:
         Associations:
           Header:
             created: Created
+            label: Pipelines currently using "{profile}"
             last24hrsnodehr: Last 24 hrs node hours
             last24hrsruns: Last 24 hrs runs
             name: Name
             namespace: Namespace
+            noAssociations: This profile is not associated with any schedules or triggers
```

Comment:
need to fix this message. This section is not only showing schedules and triggers

Reply:
The title of the section is ``Pipelines currently using this profile``. If that's the case, I think the message should be ``No pipelines currently using this profile``, no?